packages/client-mobile/ios/quollclientmobile.xcodeproj/project.pbxproj

```diff
@@ -12,9 +12,9 @@
 		13B07FBC1A68108700A75B9A /* AppDelegate.mm in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB01A68108700A75B9A /* AppDelegate.mm */; };
 		13B07FBF1A68108700A75B9A /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 13B07FB51A68108700A75B9A /* Images.xcassets */; };
 		13B07FC11A68108700A75B9A /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
+		6132EF182BDFF13200BBE14D /* PrivacyInfo.xcprivacy in Resources */ = {isa = PBXBuildFile; fileRef = 6132EF172BDFF13200BBE14D /* PrivacyInfo.xcprivacy */; };
 		7699B88040F8A987B510C191 /* libPods-quollclientmobile-quollclientmobiletests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 19F6CBCC0A4E27FBF8BF4A61 /* libPods-quollclientmobile-quollclientmobiletests.a */; };
 		81AB9BB82411601600AC10FF /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 81AB9BB72411601600AC10FF /* LaunchScreen.storyboard */; };
-		8DBFCBAE2B413CA200053547 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 8DBFCBAD2B413CA200053547 /* MaterialIcons.ttf */; };
```

Comment:
Was seeing this error so removed this as suggested [here](https://stackoverflow.com/a/73198063)
<img width="1419" alt="image" src="https://github.com/user-attachments/assets/e73e9714-482f-4e64-a87e-0ea22bd85705">